iothub/service/src/MessageSystemPropertyNames.cs

```diff
@@ -36,6 +36,9 @@ public static class MessageSystemPropertyNames
         /// </summary>
         public const string To = "to";
 
+        /// <summary>
+        /// The maximum number of times a message can transition between the Enqueued and Invisible states.
```

Comment:
Did they mention what is invisible state? Just wondering.

Reply:
I'm pretty sure the invisible state is for avoiding race conditions. Basically the message will not be available for any action as long as it is invisible.